rust/Cargo.toml

```diff
@@ -1,11 +1,20 @@
 [package]
-name = "rust"
+name = "bench"
 version = "0.1.0"
 edition = "2021"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
 [dependencies]
 memcache = "0.17.0"
 memcached-rs = "0.4.2"
 rand = { version = "0.8.5", features = ["small_rng"] }
+
+[profile.release]
+debug = 1
+
+[lib]
+name = "client"
+path = "src/client.rs"
+
+[[bin]]
```

Comment:
does this need to brackkeys?

Reply:
I don't think it's _required_ but it's a convention I've seen [[[docs](https://doc.rust-lang.org/cargo/reference/cargo-targets.html#configuring-a-target)]]

Reply:
oh in the doc it says that its https://toml.io/en/v1.0.0-rc.3#array-of-tables which means you can have multiple bins but only one lib